psi4/src/psi4/libscf_solver/sad.cc

```diff
@@ -167,35 +167,42 @@ SharedMatrix SADGuess::form_D_AO() {
 
     // Spin occupations per atom, to be determined by Hund's Rules
     // or user input
-    std::vector<int> nalpha(molecule_->natom(), 0);
-    std::vector<int> nbeta(molecule_->natom(), 0);
+    std::vector<double> nalpha(molecule_->natom(), 0);
+    std::vector<double> nbeta(molecule_->natom(), 0);
     std::vector<int> nelec(molecule_->natom(), 0);
-    std::vector<int> nhigh(molecule_->natom(), 0);
-    int tot_elec = 0;
 
     // Ground state high spin occupency array, atoms 0 to 36 (see Giffith's Quantum Mechanics, pp. 217)
     // For 37 to 86, save for f-block: Atomic, Molecular, & Optical Physics Handbook, Ed. Gordon W. F. Drake, American
     // Institute of Physics, Woodbury, New York, USA, 1996.
-    const int reference_S[] = {0, 1, 0, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1, 2, 3, 6, 5, 4, 3, 2,
-                               1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1, 2, 5, 6, 5, 4, 3, 0, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1,
-                               0, 3, 4, 5, 6, 7, 8, 5, 4, 3, 2, 1, 0, 1, 2, 3, 4, 5, 4, 3, 2, 1, 0, 1, 2, 3, 2, 1, 0};
-    const int MAX_Z = 86;
+    const std::vector<int> reference_S = {0, 1, 0, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1,
+                                          2, 3, 6, 5, 4, 3, 2, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1, 2, 5, 6, 5,
+                                          4, 3, 0, 1, 0, 1, 2, 3, 2, 1, 0, 1, 0, 1, 0, 3, 4, 5, 6, 7, 8, 5,
+                                          4, 3, 2, 1, 0, 1, 2, 3, 4, 5, 4, 3, 2, 1, 0, 1, 2, 3, 2, 1, 0};
```

Comment:
Would you mind reformatting as
```
// clang-format off
const std::vector<int> reference_S = { 0,
1,                                                                                           0,
1, 0,                                                                         1, 2, 3, 2, 1, 0,
1, 0,                                                                         1, 2, 3, 2, 1, 0,
1, 0,                                           1, 2, 3, 6, 5, 4, 3, 2, 1, 0, 1, 2, 3, 2, 1, 0,
1, 0,                                           1, 2, 5, 6, 5, 4, 3, 0, 1, 0, 1, 2, 3, 2, 1, 0, 
1, 0, 1, 0, 3, 4, 5, 6, 7, 8, 5, 4, 3, 2, 1, 0, 1, 2, 3, 4, 5, 4, 3, 2, 1, 0, 1, 2, 3, 2, 1, 0 };
// clang-format on
```
for readability, please?

Reply:
Done, thanks!